config/spec/defaults.go

```diff
@@ -57,8 +57,10 @@ const (
 	defaultTargetSecondsPerEth1Block = 2 // Berachain specific.
 
 	// Fork-related values.
-	defaultDeneb1ForkEpoch  = 9999999999999998 // Set as a future epoch as not yet determined.
-	defaultElectraForkEpoch = 9999999999999999 // Set as a future epoch as not yet determined.
+	defaultDeneb1ForkEpoch                 = 9999999999999998 // Set as a future epoch as not yet determined.
+	defaultElectraForkEpoch                = 9999999999999999 // Set as a future epoch as not yet determined.
+	defaultInactivityPenaltyQuotient       = 67108864         // Phase 0
+	defaultInactivityPenaltyQuotientAltair = 50331648         // Altair
```

Comment:
These values are taken from ETH2.0 consensus spec.

Reply:
cc @calbera I think our current fork is Deneb1 already, so there should be no need to add defaults here?

Reply:
I see, @calbera could you pls confirm.

Reply:
My bad, Thanks for clarifying. I have hardcoded the values in node-api/backend/spec.go with comments. And removed from here.